modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/js/WalkableSample.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {Walkthrough} from 'frontend-js-components-web';
+import React from 'react';
+
+const CURRENT_PAGE_MOCK = {
+	closeOnClickOutside: false,
+	closeable: true,
+	skippable: true,
+	steps: [
+		{
+			content: '<span>Content 1</span><br/><code>Hello1</code>',
+			darkbg: true,
```

Comment:
I took this example API from https://liferay.atlassian.net/wiki/spaces/ENGLEXICON/pages/1995737164/DESIGN-664+Walkthrough#JSON-format-example%3A and I have a question.

What darkbg does? Does it applies https://liferay.atlassian.net/wiki/spaces/ENGLEXICON/pages/1995737164/DESIGN-664+Walkthrough#Highlighted-Area? 

Currently on Clay side we can have only the darken overlay not the box-shadow based one(this one can be painful since the container area of the elements doesn't have the same size as the docs describes 🤔 )

Reply:
/cc @marcoscv-work @dsanz @drakonux

Reply:
The highlighted area (or elements) should be on top of the dark background. But, probably I didn't understand very well the question @diegonvs

Reply:
I think `darkbg` describves the behavior of the what's behind the popover. Design says that it can be dark or transparent.

I haven't defined that property (I believe @marcoscv-work did), but my interpretation is as follows.
> What darkbg does? Does it applies https://liferay.atlassian.net/wiki/spaces/ENGLEXICON/pages/1995737164/DESIGN-664+Walkthrough#Highlighted-Area?

Yes

- darkbg: false → use a transparent background behind the popover
- darkbg: true → use a dark background behind the popover

Let's wait for confirmation